chain-impl-mockchain/src/transaction/transfer.rs

```diff
@@ -92,6 +94,13 @@ impl Input {
         }
     }
 
+    pub fn from_account_pk(public_key: SpendingPublicKey, value: Value) -> Self {
```

Comment:
The abbreviation is a bit short unless the same is used everywhere.
`from_account_public_key`?